thunder.post_update.php

```diff
@@ -22,3 +24,81 @@ function thunder_post_update_ensure_config_selector_installed() {
   // Output logged messages to related channel of update execution.
   return $thunderUpdater->logger()->output();
 }
+
+/**
+ * Installs the content_content_lock view.
+ */
+function thunder_post_update_add_content_lock_view() {
+
+  $content_view = View::load('content');
+  if (empty($content_view)) {
+    return;
+  }
+  $content_view->setThirdPartySetting('config_selector', 'feature', 'thunder_content_view');
+  $content_view->setThirdPartySetting('config_selector', 'priority', 0);
+
+  $view_contains_content_lock = FALSE;
+  foreach (array_keys($content_view->get('display')) as $display_name) {
+    $display = $content_view->getDisplay($display_name);
+    $options = ['fields', 'filters', 'sorts', 'relationships'];
+    foreach ($options as $option) {
+      if (!empty($display['display_options'][$option]) && is_array($display['display_options'][$option])) {
+        foreach ($display['display_options'][$option] as $key => $values) {
+          if ($values['table'] == 'content_lock') {
+            $view_contains_content_lock = TRUE;
```

Comment:
Rather than doing this we could just calculate dependencies and check if content_lock module is there no?

Reply:
Good point 👍